### DISCUSSION TITLE
Sound ok, but video looks low quality and image changes only every 5-10 seconds

### DISCUSSION THREAD

**Discussion by alphanet72** [goggle/plugin.video.rtsplaytv#17] [General]
Hello,

OSMC [2024.08-1](https://ftp.fau.de/osmc/osmc/download//installers/diskimages/OSMC_TGT_rbp4_20240825.img.gz)
on a raspi4 4 GB.

Arte+7 seems to work quite well (good quality sound&video).

RTS Play TV has good sound, but video quality and it's a bit like a slide-show (the image changes only every 5-10 seconds).
Changing the option about HD quality does not change anything.
Plugin version: 2.2.4

Do you have an idea?  What should I try?

Thank you.

**Comment by alphanet72** [comment]
This only happened with the very latest Telejournal.  Did not happen with the other broadcasts or older version of the Telejournal.  Maybe there is something funny with the "live" version.

**Comment by goggle** [comment]
Unfortunately, I have no idea what the issue could be. I couldn't reproduce this issue. It somewhat sounds like your Pi could be overloaded, but honestly, I don't really have a clue. Does the issue still persist?

**Comment by alphanet72** [comment]
Hello,

On Tue, Oct 01, 2024 at 03:30:27PM -0700, Alexander Seiler wrote:
> Unfortunately, I have no idea what the issue could be. I couldn't reproduce this issue. It somewhat sounds like your Pi could be overloaded, but honestly, I don't really have a clue. Does the issue still persist?

Thank your for trying to reproduce it.

I will also try on another raspi4, and maybe with different versions of kodi and OSMC.

Have a nice day.

**Comment by alphanet72** [comment]
Hello again,

On Wed, Oct 02, 2024 at 08:39:31AM +0200, Marc SCHAEFER wrote:
> On Tue, Oct 01, 2024 at 03:30:27PM -0700, Alexander Seiler wrote:
> > Unfortunately, I have no idea what the issue could be. I couldn't reproduce this issue. It somewhat sounds like your Pi could be overloaded, but honestly, I don't really have a clue. Does the issue still persist?
> 
> Thank your for trying to reproduce it.
> 
> I will also try on another raspi4, and maybe with different versions of kodi and OSMC.

I think I have located the issue: some of the RTS play media are protected
through a proprietary codec.

For those media, the first time, WineCDM is loaded automatically.

However, then, when playing, you get the "slide show" effect: sound is ok, but
the image only changes every 5-10 seconds.

For example, on RTS, "Les enquêtes de Murdoch" seem to be protected.

I guess you don't do anything about it and the issue is really the kodi video
playback using WineCDM?

Searching for this specific issue did not bring much, I am struggling for what
keyword to use for "video does not play like a stream, but only get one image,
which sticks for 5-10 seconds, then another image".

Maybe it's MPEG and we only get to see the I (complete) image, and the P and B
images don't get decoded?

Have a nice night.

**Comment by goggle** [comment]
Thanks for mentioning a video. I have tried it and can confirm the issue (even on my PC, so it is definitely not a Pi issue). It only appears when playing a DRM protected video. I know that is used to work correctly when I first implemented it, although the video quality was always quite low (resolution-wise), but it definitely didn't have this slide-show effect. I'm not sure if the problem is a plugin issue or it is more a issue of Kodi itself.

I have opened an issue for this problem: https://github.com/goggle/script.module.srgssr/issues/30
